docs/source/api/apollo-server.mdx

```diff
@@ -199,6 +199,21 @@ An array containing custom functions to use as additional [validation rules](htt
 <tr>
 <td>
 
+###### `disableValidation`
```

Comment:
I'd consider giving this a name like `dangerouslyDisableValidation` (like React — I'm not sure if we have examples of similar names in Apollo Server or other Apollo projects already), and give more context on potential side effects. My vague recollection is that graphql-js execution (and perhaps aspects of Apollo Server like the plugin system or usage reporting) does assume that some of the validations have been run and that this can have highly unexpected effects at runtime (certainly including security issues if you are assuming that values have expected types).

Reply:
Sure - I'm fine with that

Reply:
Updated